_tips/designing.ja.md

```diff
@@ -0,0 +1,669 @@
+  <figure class="reference pass">
+    <figcaption>{% include_cached icon.html name="check-circle" label="OK" %}色と記号を使った場合</figcaption>
+    <div>
+      <form action="#" method="post">
+        <p class="instruction"><span class="required" style="color: #992929;">必要な箇所は</span> 赤で記述され <span class="required" style="color: #992929;">*</span>の記号とともに表示</p>
+        <div class="row">
+          <label for="name2">名前</label> <input type="text" id="name2" name="name"><br>
+        </div>
+        <div class="row">
+          <label class="required" style="color: #992929;" for="email2">メール *</label> <input type="text" id="email2" name="email">
+        </div>
+      </form>
+    </div>
+  </figure>
+</div>
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="例: 色だけで何かを参照する場合" class="example" %}
+{:/}
+<div class="color-alone two-column">
+  <figure class="question fail">
+    <figcaption>{% include_cached icon.html name="ex-circle" label="Wrong" %}色だけの場合</figcaption>
+    <div>
+      <svg version="1.1" width="153" height="154" aria-labelledby="inaccessible-triangles-title" role="img">
+        <g>
+          <title id="inaccessible-triangles-title">異なる角度を持つ4つの三角形</title>
+          <path d="m2.06902,49.92402l91.82715,-48.2264l-42.35869,75.85164l-49.46846,-27.62524z" fill="#09610D"/>
+          <path d="m147.18413,95.39555l-41.64554,-80.62148l43.24728,27.76369l-1.60175,52.85779z" fill="#0000CC"/>
+          <path d="m110.9136,148.01692l-27.57392,-82.3039l69.97661,50.70448l-42.40269,31.59943z" fill="#AA0402"/>
+          <path d="m21.07534,112.17733l48.69831,-22.53485l-4.90304,52.17411l-43.79528,-29.63926z" fill="#DDDD00"/>
+        </g>
+      </svg>
+      <form action="#" method="post">
+        <fieldset>
+          <legend style="width: 100%;">どれが直角三角形か?</legend>
+          <input id="answer-green1" name="answer1" value="green" type="radio"> <label for="answer-green1">緑</label><br>
+          <input id="answer-blue1" name="answer1" value="blue" type="radio"> <label for="answer-blue1">青</label><br>
+          <input id="answer-red1" name="answer1" value="red" type="radio"> <label for="answer-red1">赤</label><br>
+          <input id="answer-yellow1" name="answer1" value="yellow" type="radio"> <label for="answer-yellow1">黄色</label><br>
+          <input id="answer-pass1" name="answer1" value="pass" type="radio"> <label for="answer-pass1">わからない</label>
+        </fieldset>
+      </form>
+    </div>
+  </figure>
+  <figure class="question pass">
+    <figcaption>{% include_cached icon.html name="check-circle" label="OK" %}色と数字</figcaption>
+    <div>
+      <svg version="1.1" width="153" height="154">
+        <g>
+          <path d="m2.06902,49.92402l91.82715,-48.2264l-42.35869,75.85164l-49.46846,-27.62524z" fill="#09610D" />
+          <text fill="#f4f4f4" y="51" x="44"><tspan>1</tspan><tspan dx="100" style="opacity:0">Green triangle: Angles are 30°, 60°, 90°</tspan></text>
```

Comment:
@jaykishigami Please translate the text in this line: `Green triangle: Angles are 30°, 60°, 90°` and the following lines.